assets/src/components/routeVariantName.tsx

```diff
@@ -17,7 +17,7 @@ export const RouteVariantName = ({
   return (
     <output
       aria-label="Route Variant Name"
-      className={"m-route-variant-name " + className}
+      className={"c-route-variant-name " + className}
```

Comment:
**issue (non-blocking):** I actually just noticed while QA-ing that `className` can actually be undefined, meaning that we put the literal string `"undefined"` here. If you want to fix it quickly that could be nice but if you feel like it's out of scope for this PR that's fine too.